src/SFA.DAS.IdentifyDataLocks.IntegrationTests/SFA.DAS.IdentifyDataLocks.IntegrationTests.csproj

```diff
@@ -12,6 +12,9 @@
     <None Remove="TestData\Dlock01_DataLocks.json" />
     <None Remove="TestData\Dlock01_Earnings.json" />
     <None Remove="TestData\EarningEvents.json" />
+    <None Remove="TestData\LearnerWithNullData.json" />
```

Comment:
Perhaps this is not required as I see you have added a set of files with in `stopped` folder, may these needs replacing with new set?